src/main/java/ru/practicum/shareit/item/dao/ItemStorageImpl.java

```diff
@@ -0,0 +1,93 @@
+package ru.practicum.shareit.item.dao;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.user.dao.UserStorage;
+import ru.practicum.shareit.user.model.User;
+
+import java.util.HashMap;
+import java.util.Map;
+
+@Service
+@RequiredArgsConstructor
+public class ItemStorageImpl implements ItemStorage {
+    private final UserStorage userStorage;
+
+    private HashMap<Integer, Item> items = new HashMap<>();
```

Comment:
1) Может быть [final](https://vertex-academy.com/tutorials/ru/modifikator-final/)
2) Лучше обозначать интерфейс, а уже инициализировать объект с определенной реализацией. Так программа становится более гибкой и не сломается, если нужно будет подменить реализацию на другую
```suggestion
    private Map<Integer, Item> items = new HashMap<>();
```